<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NUFEB Designer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PaintingWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Width (um)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>Layout and Assign Taxa</string>
    </property>
    <widget class="QPushButton" name="assignTaxaProportionButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Specify Proportions...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="distributeRandomButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>96</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Random</string>
     </property>
    </widget>
    <widget class="QPushButton" name="distributePoissonButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Poisson</string>
     </property>
    </widget>
    <widget class="QPushButton" name="distributeGridButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Num Bacteria</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Rows</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="randomNumBactText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="poissonRadiusText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="gridRowsText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="gridColsText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="assignTaxaEvenlyButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Evenly</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign Taxa to Unassigned: </string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="clearTaxaAssignmentsButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Taxa Assignment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearDrawingButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Points</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Length(um)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ceiling (um)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genProjectButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>590</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate NUFEB Inputs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthText">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ceilText">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>View Scale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="viewScaleText">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="applyDimensions">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editTaxaButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load/Edit Taxa...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>680</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sim</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="runTimeText">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>680</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>7200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="simTimeUnits">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>680</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seconds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Minutes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hours</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Days</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>710</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seedText">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>710</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1701</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInteractions">
    <property name="title">
     <string>Interactions</string>
    </property>
    <addaction name="actionType_VI_Secretion"/>
   </widget>
   <widget class="QMenu" name="menuTaxa_Manager">
    <property name="title">
     <string>Taxa Manager</string>
    </property>
    <addaction name="actionView_Available"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Edit_Taxa"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInteractions"/>
   <addaction name="menuTaxa_Manager"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionType_VI_Secretion">
   <property name="text">
    <string>Type VI Secretion...</string>
   </property>
   <property name="iconText">
    <string>Type VI Secretion...</string>
   </property>
   <property name="toolTip">
    <string>Type VI Secretion...</string>
   </property>
  </action>
  <action name="actionView_Available">
   <property name="text">
    <string>Go to Taxa Library...</string>
   </property>
   <property name="iconText">
    <string>Go to Taxa Library...</string>
   </property>
   <property name="toolTip">
    <string>Go to Taxa Library...</string>
   </property>
  </action>
  <action name="actionCreate_New_Taxon">
   <property name="text">
    <string>Create New Taxon</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionLoad_Edit_Taxa">
   <property name="text">
    <string>Load/Edit Taxa...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintingWidget</class>
   <extends>QGraphicsView</extends>
   <header>PaintingWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>add_taxa_to_list()</slot>
 </slots>
</ui>
